mattermost-plugin/webapp/src/components/rhsChannelBoardItem.tsx

```diff
@@ -47,6 +47,7 @@ const RHSChannelBoardItem = (props: Props) => {
 
     const untitledBoardTitle = intl.formatMessage({id: 'ViewTitle.untitled-board', defaultMessage: 'Untitled board'})
 
+    const html = Utils.htmlFromMarkdown(board.description)
```

Comment:
nit: I don't like the variable name `html`. Change to something more descriptive. ie. `markdownHtml`

Reply:
Make sense, will change it.